### DISCUSSION TITLE
bowtie2のインストール方法

### DISCUSSION THREAD

**Discussion by oogasawa** [nig-sc/nigsc_homepage2#280] [Q&A] [answered]
(sc-helpdeskへの質問を質問者の許可を得て代理で起票しております)

bowtie2と呼ばれるアプリケーションのインストール方法についてお伺いしたいと存じます。まず、初めにminicondaをスパコンにインストールし、その後qloginをし、bowtie2のcondaを介したインストール方法を試みました。

 conda install -c bioconda bowtie2

色々と画面が出てくるのですが、bowtie2 あるいはbowtie2 --versionなどとタイプしてもcommand not
foundと出ますので、おそらくインストールされていないように思います。
bowtie2はcondaを通してのインストールは出来ないとの認識で正しいでしょうか。
また、インストール方法をご存じであればご教示いただけますと幸甚でございます。
何卒よろしくお願い申し上げます。

**Comment by motthy** [comment, marked as answer]
遺伝研スパコンで、miniconda3環境下での

`conda install -c bioconda bowtie2`

を試してみましたが、問題なくインストールされてヘルプメッセージも表示できました。
「いろいろと画面が出てくる」ところで、エラーメッセージなど表示されているでしょうか？
それを拝見できれば問題が切り分けられるかと...

**Reply by motthy** [reply]
なお、miniconda3でバイオインフォマティクス関連のツールをインストールする際には、仮想環境を作ってそこにインストールすることをお勧めします。（もしインストールがうまくいかなかった場合は仮想環境ごと削除）

bowtie2という名前で仮想環境を作成する場合、
```
conda create -n bowtie2
conda activate bowtie2
conda install -c bioconda bowtie2
```
というコマンドを順番に実行します。

**Reply by EGFR1** [reply]
お返事いただき有難うございます。
仮想環境設定でインストールが出来ました。
迅速なご対応、感謝申し上げます。

**Reply by motthy** [reply]
よかったです！
実行するときには仮想環境への切り替えをお忘れなく
（スクリプトで実行するときにも　`conda activate bowtie2`　の記載をお忘れなく）
